Release notes for Red Hat Developer Hub 1.4- Update Release Note Texts (#758)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/snip-bug-fix-rhidp-2903.adoc

[id="bug-fix-rhidp-2903"]

> = All API documentation now defined in the 3scale backend plugin

Reviewer: ```suggestion
= All API documentation is defined in the 3scale backend plugin
```

> Before this update, not all API documentation defined in the 3scale backend plugin were available in the backstage.

Reviewer: ```suggestion
Previously, some API documentation defined in the 3scale backend plugin was not accessible in {product-very-short}.
```

> With this update, all API documentation defined in the 3scale backend plugin are imported and merged in the backstage.

Reviewer: ```suggestion
With this update, all API documentation defined in the 3scale backend plugin is imported and merged in the {product-very-short}.
```
